src/I18n/TranslatorRegistry.php

```diff
@@ -155,6 +169,35 @@ public function defaultFormatter($name = null)
     }
 
     /**
+     * Set fallback domain. Messages from the fallback domain are used if
+     * a domain can't provide a localization message.
+     *
+     * Get the currently set domain if argument is null.
+     *
+     * @param string|null $name fallback domain
+     * @return string
+     */
+    public function defaultFallbackDomain($name = null)
+    {
+        if ($name === null) {
+            return $this->_defaultFallbackDomain;
+        }
+
+        return $this->_defaultFallbackDomain = $name;
+    }
+
+    /**
+     * Set if the default domain fallback is used.
+     *
+     * @param bool $enable flag to enable or disable fallback
+     * @return void
+     */
+    public function useTranslationFallback($enable = true)
```

Comment:
`useFallback()` would be shorter, and this class is already scoped to translations.